apis/nucleus/src/index.js

```diff
@@ -446,7 +447,8 @@ function nuked(configuration = {}) {
             root.remove(api._popoverInstance);
             api._popoverInstance = null;
           }
-          const opts = getListboxPopoverOptions(options);
+          const onPopoverClose = () => this.emit('closePopover');
```

Comment:
so we can catch this in the talbe?

Reply:
we will listen to this event, then set the closing state to false in order to keep track of the popover state to pass it for visibility of ... menu

Reply:
```suggestion
          const onPopoverClose = () => api.emit('fieldPopoverClose');
```